.idea/misc.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
.idea müssten wir mal in gitignore packen (müsste laut google von "intellij" kommen)